### DISCUSSION TITLE
Clarify license

### DISCUSSION THREAD

**Discussion by jirutka** [ada-url/ada#326] [Q&A] [answered]
Can you please clarify how is ada licensed? There are LICENSE-APACHE and LICENSE-MIT files in the repo, so is it dual-licensed (`Apache-2.0 OR MIT`) or some parts are licensed under Apache-2.0 and some under MIT (`Apache-2.0 AND MIT`)?

**Comment by lemire** [comment, marked as answer]
@anonrig  Please review https://github.com/ada-url/ada/pull/328

**Reply by anonrig** [reply]
@jirutka Hope that answers your question.

**Reply by jirutka** [reply]
Yes, thank you!
